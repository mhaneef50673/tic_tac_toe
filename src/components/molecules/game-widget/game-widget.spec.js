import React from "react";
import { mount } from "enzyme";
import { mountToJson } from "enzyme-to-json";
import GameWidget from "./";

describe("onGameOver Component", () => {
  it("should render the component", () => {
    const tree = mount(<GameWidget onGameOver={() => {}} />);
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});
