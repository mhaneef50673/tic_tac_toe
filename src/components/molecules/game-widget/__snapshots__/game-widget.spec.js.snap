// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`onGameOver Component should render the component 1`] = `
<GameWidget
  onGameOver={[Function]}
>
  <div
    className="game-widget"
  >
    <div
      className="panel-wrapper"
    >
      <div
        className="panel-container"
      >
        <h2>
          Game
        </h2>
        <div
          className="game-widget_container"
        >
          <Row
            noGutters={false}
          >
            <div
              className="row"
            >
              <Col>
                <div
                  className="col"
                >
                  <GameForm
                    isError={false}
                    onGameStartHandler={[Function]}
                    onInputChange={[Function]}
                    playerOne=""
                    playerTwo=""
                    shouldStartGame={false}
                  >
                    <form
                      onSubmit={[Function]}
                    >
                      <label
                        className="input-label"
                      >
                        Player one Name
                      </label>
                      <input
                        className="input-text"
                        onChange={[Function]}
                        placeholder="Enter player one name"
                        required={true}
                        type="text"
                        value=""
                      />
                      <label
                        className="input-label"
                      >
                        Player two Name
                      </label>
                      <input
                        className="input-text"
                        onChange={[Function]}
                        placeholder="Enter player two name"
                        required={true}
                        type="text"
                        value=""
                      />
                      <div
                        className="btn-center "
                      >
                        <button
                          className="btn-class"
                          type="submit"
                        >
                          Start Game
                        </button>
                      </div>
                    </form>
                  </GameForm>
                </div>
              </Col>
            </div>
          </Row>
        </div>
      </div>
    </div>
  </div>
</GameWidget>
`;
